cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -1752,8 +1752,18 @@
           "additionalProperties": false,
           "properties": {
             "enabled": {
-              "type": ["boolean", "string"],
-              "description": "Whether vendor data is enabled or not. Use of string for this value is DEPRECATED. Default: ``true``"
+              "description": "Whether vendor data is enabled or not. Default: ``true``",
```

Comment:
Schema looks good, but documentation loses the DEPRECATED information from the description. I wonder if we can accumulated all oneOf descriptions (or we we just ignore the `deprecated: true` items and don't document that type

I don't like docs representing the "string" type, but not telling about deprecation. We should either redact string type or find a way to accumulate all description strings.
```diff
$ diff -urN docs.main docs.scriptsvendor 
--- docs.main	2022-08-03 09:55:10.217744060 -0600
+++ docs.scriptsvendor	2022-08-03 10:03:46.331339662 -0600
@@ -59,7 +59,7 @@
 **Config schema**:
     **vendor_data:** (object)
 
-        **enabled:** (boolean/string) Whether vendor data is enabled or not. Use of string for this value is DEPRECATED. Default: ``true``
+        **enabled:** (boolean/string) Whether vendor data is enabled or not. Default: ``true``
 
         **prefix:** (string/array of (string/integer)) The command to run before any vendor scripts. Its primary use case is for profiling a script, not to prevent its run
```

Reply:
Coming in #1636.